README.md

```
@@ -123,4 +123,60 @@ java -jar build/libs/WCS_DataStream-*.jar
+
+### 서비스 실행 순서
+1) Zookeeper → 2) Kafka → 3) Redis → 4) PostgreSQL → 5) 애플리케이션 → 6) Prometheus → 7) Grafana
+
+### 서비스 수량(기본 도커 컴포즈)
+- App 1, PostgreSQL 1, Redis 1, Zookeeper 1, Kafka 1, Kafka-UI 1, Prometheus 1, Grafana 1, (선택) Exporters 2 
+
+## 환경 구성 표
+
+| 구분 | 항목 | WCS_DataStream |
+|---|---|---|
+| 개발 Framework | Base | Spring Boot 3.5.3 ||  | Front-End | 정적 HTML(대시보드) / Grafana |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 3; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)